projects/plugins/jetpack/modules/search/instant-search/components/search-filters.jsx

```diff
@@ -60,20 +60,22 @@ class SearchFilters extends Component {
 			/>
 		);
 
-	renderStaticFilterComponent = ( configuration ) => {
+	renderStaticFilterComponent = configuration => {
```

Comment:
Are the changes to the 5 instant search module files supposed to be in this PR?

Reply:
these changes are all from the eslint update. When i revert them they re-apply on the next commit.

Reply:
Though, at this point, it doesn't really matter. Fine to have them land.